tasks/configure.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
everything here is copy-paste from old main.yml excluding last part with sql importing